src/chains/ethereum/ethereum/tests/api/eth/getTransactionReceipt.test.ts

```diff
@@ -0,0 +1,74 @@
+import getProvider from "../../helpers/getProvider";
+import assert from "assert";
+import EthereumProvider from "../../../src/provider";
+
+describe("api", () => {
+  describe("eth", () => {
+    describe("getTransactionReceipt", () => {
+      let provider: EthereumProvider;
```

Comment:
Eventually we'll port more `eth_getTransactionReceipt` tests over, so let's add another `describe` block here. Something like

```typescript
describe("legacy instamine detection and warning", () => {
  let provider: EthereumProvider;
  // ...
});
  ```